webpack.config.node.js

```diff
@@ -28,6 +29,9 @@ const bundleEnv = config( 'env' );
 const isDevelopment = bundleEnv === 'development';
 
 const commitSha = process.env.hasOwnProperty( 'COMMIT_SHA' ) ? process.env.COMMIT_SHA : '(unknown)';
+const buildTimestamp = moment()
```

Comment:
Would (new Date()).toISOString() be a valid alternative to pulling in moment?

Reply:
+1. The less moment usage, the better.

Reply:
Fixed in 146279349863b425e5f6bfb7fa0a1b8875cfad5f!